utils/user_template.py

```diff
@@ -1,7 +1,16 @@
 import json
+import spotipy
+from utils.auth import get_auth_manager
+
 
 # TODO: Create the sample template here when the pin functionality is added.
-user_template = {}
+auth_manager = get_auth_manager()
+spotify = spotipy.Spotify(auth_manager=auth_manager)
+spotify_user = spotify.current_user()
+
+user_template = {
+    "username": spotify_user,
+}
```

Comment:
As per our discussion on Discord, seems like we don't need to store the user information. Please remove this.